noobaa_sa/account.py

```diff
@@ -106,7 +120,7 @@ def create(self, account_name, access_key, secret_key, config_root=None):
         retcode, stdout, stderr = self.conn.exec_cmd(cmd)
         if retcode != 0:
             raise AccountCreationFailed(
-                f"Creation of account failed with error {stderr}"
+                f"Creation of account failed with error {stdout}"
```

Comment:
error stored in stdout or stderr ?

Reply:
Before adding the creation_date property as a workaround for https://bugzilla.redhat.com/show_bug.cgi?id=2260325, I found the error message from stdout to be a lot clearer and helpful than stderr:

stderr (seems to show the last logs that were written to the same source of `journalctl -u noobaa_nsfs`):
```
E           noobaa_sa.exceptions.AccountCreationFailed: Creation of account failed with error load_nsfs_nc_config.setting config.NSFS_NC_CONF_DIR <config_root>
E           config.load_nsfs_nc_config could not find config.json... skipping
E           (node:52447) NOTE: We are formalizing our plans to enter AWS SDK for JavaScript (v2) into maintenance mode in 2023.
E           Please migrate your code to use AWS SDK for JavaScript (v3).
E           For more information, check the migration guide at https://a.co/7PzMCcy
E           (Use `node --trace-warnings ...` to show where the warning was created)
```

stdout:
```
E           noobaa_sa.exceptions.AccountCreationFailed: Creation of account failed with error {
E             "error": {
E               "code": "InvalidSchema",
E               "message": "Schema invalid, please use required properties",
E               "detail": "Error: must have required property 'creation_date'\n    at Object.validate_account_schema (/usr/local/noobaa-core/src/manage_nsfs/nsfs_schema_utils.js:37:23)\n    at add_account (/usr/local/noobaa-core/src/cmd/manage_nsfs.js:526:23)\n    at async manage_account_operations (/usr/local/noobaa-core/src/cmd/manage_nsfs.js:619:9)\n    at async account_management (/usr/local/noobaa-core/src/cmd/manage_nsfs.js:391:5)\n    at async main (/usr/local/noobaa-core/src/cmd/manage_nsfs.js:191:13)"
E             }
E           }
```